src/clojure/com/interrupt/ibgateway/component/figwheel.clj

```diff
@@ -2,7 +2,7 @@
   (:require [mount.core :refer [defstate] :as mount]
             [figwheel-sidecar.repl-api :as ra]))
 
-#_(defstate figwheel
+(defstate figwheel
```

Comment:
Why do we need both this and `com.interrupt.ibgateway.component.figwheel.figwheel`?

Reply:
Good question. I'm still trying to have a figwheel service. But that would have to connect to a second repl, if we want both a clojure and clojurescript repl.